test/common.py

```diff
@@ -87,6 +87,9 @@ def get_gpu_type(t):
 def to_gpu(obj, type_map={}):
     if torch.is_tensor(obj):
         t = type_map.get(type(obj), get_gpu_type(type(obj)))
+        # Workaround since torch.HalfTensor doesn't support clone()
+        if type(obj) == torch.HalfTensor:
+            return obj.new().resize_(obj.size()).copy_(obj).type(t)
```

Comment:
Honestly, we should just add `clone()` support to `HalfTensor`, even if it is just this polyfill :)

Reply:
Yea good idea, I will look into adding this hack

Reply:
Discussed with @ezyang offline and we agreed not to add this method for now.